src/QmlControls/QGCCheckBox.qml

```diff
@@ -9,6 +9,21 @@ CheckBox {
     property var __qgcPal: QGCPalette { colorGroupEnabled: enabled }
 
     style: CheckBoxStyle {
+        indicator: Rectangle {
+            implicitWidth: 16
+            implicitHeight: 16
+            radius: 3
+            border.color: control.activeFocus ? "darkblue" : "gray"
+            border.width: 1
+            Rectangle {
+                visible: control.checked
+                color: "#555"
+                border.color: "#333"
+                radius: 1
+                anchors.margins: 4
+                anchors.fill: parent
+            }
+        }
```

Comment:
I looked at the OSX implementation of Base for indicator and it has way more stuff than just this. For example the checkmark image plus lots of other complexity. How is this code setting the checkmark bitmap which for OS X is part of the indicator style.